### DISCUSSION TITLE
Hjson support

### DISCUSSION THREAD

**Discussion by crizzis** [npm/rfcs#561] [RRFC]
Hi! I was wondering how you would feel about the idea of introducing first-class support for a `package.hjson` configuration file, as a way of bringing the ability to put comments inside the config file (among other syntactic tweaks). 

(suggesting a `package.js` would have been more natural, of course, but I imagine there are good reasons why you would not want the configuration file to be a source file)

**Comment by ljharb** [comment]
https://github.com/npm/rfcs/discussions/230 has an explanation of why not, and that’s for json5, which is a massively more popular format.
